### DISCUSSION TITLE
blog/free-certificates-with-certbot/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#341] [Announcements]
# blog/free-certificates-with-certbot/

实测推荐使用 Certbot 为网站设置永久免费的 HTTPS 证书，超简单，全程不用五分钟！

https://jimmysong.io/blog/free-certificates-with-certbot/

**Comment by LHiaoeng** [comment]
windows系统 nginx如何设置？

**Reply by rootsongjc** [reply]
可以参考下这篇文章：[使用 Nginx 结合 CertBot 配置 HTTPS 协议
](https://developer.aliyun.com/article/689901)
